crates/model/src/order.rs

```diff
@@ -493,6 +493,27 @@ pub struct EthflowData {
     pub is_refunded: bool,
 }
 
+#[serde_as]
+#[derive(Eq, PartialEq, Clone, Derivative, Deserialize, Serialize)]
+#[derivative(Debug)]
+#[serde(rename_all = "camelCase")]
```

Comment:
```suggestion
#[derive(Debug, Eq, PartialEq, Clone, Derivative, Deserialize, Serialize)]
```
I think we don't need the extra attributes. The struct below needs camelCase but not the rest either.

Reply:
`camelCase` has an effect on how the enum names get serialized/deserialized (`QuoteIdNotFound` vs `quoteIdNotFound`) so I would argue needing `serde(rename_all = ...)` depends on what the expected output is.

Agree that `serde_as` and `derivative` aren't used here and can be dropped.